src/main/java/csv/ReadCSV.java

```diff
@@ -0,0 +1,33 @@
+package main.java.csv;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadCSV {
+    private final static String PEOPLE_DATA = "people.csv";
+
+    public static void main(String[] args) {
+
+        File file = new File(PEOPLE_DATA);
+
+        if (file.exists()) {
+            try (BufferedReader bufferedReader = new BufferedReader(new FileReader(PEOPLE_DATA))) {
+                String string;
+                while ((string = bufferedReader.readLine()) != null) {
+                    System.out.println(string);
+                }
+
+            } catch (IOException e) {
+                String messageFormat = "Error: %s;";
```

Comment:
What is the difference between 'error' and 'exception'?

Reply:
Error it's a mistake, it says to us that me have a problem in code. Exception it's a part of code, that gives us insight that something went wrong beyond our code. 
Error stops code, while exception processes the problem and continue run code.

Reply:
`Error stops code`, but `exception` do the same isn't it?

`Exception it's a part of code, that gives us insight that something went wrong beyond our code.`
But error gives us insight as well.

But what is the difference in terms of Java?

Reply:
The diference between 'error' and 'exception' is that we know in which part of code can appear excepion, and know that we can handle it. 
Unlike exception, 'error' can appear in each part of code and we don't know where and when exactly it comes, so we can't handle it. Errors can comes us from outside reasons, that we can't take under control.